cmake/Dependencies.cmake

```diff
@@ -60,39 +60,3 @@ if(BUILD_TEST)
   )
   find_package(GTest REQUIRED)
 endif()
-
```

Comment:
Not sure about removing this. It was useful at times but maybe you are right that it is better to remove in the long run. SteamHPC CI may need updating. @neon60 please confirm.